gui/package_app

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash -e
+#
+# package GUI application
+#
+
+function convert_qt_translations()
+{
+# Combine the Qt translation files we use into local qt_??.qm files.
+#
+# It is recommended to combine Qts .qm files, this script does that for
+# linux and macos, windeployqt does this for windows.
+# https://doc.qt.io/qt-5/linguist-programmers.html#deploying-translations
+#
+# This script is created from the log of the windows build from windeployqt
+# with Qt 5.12.1.
+# From the log you can see which translation files are used which depends on
+# which Qt modules we use.
+# In our case these are qtbase_*.qm, qtdeclarative_*qm and qtserialport_*.qm.
+#
+# Note with Qt5 the Qt distributed qt_xx.qm files are metacatalogs, and just
+# copying or converting them won't copy the dependencies.
+
+  if [ "${machine}" = "Mac" ]; then
+    resourcedir="${APPDIR}/Contents/Resources"
+
+    # caution, mktemp here is macos specific version
+    resourceskel="$(mktemp -t gpsbabel_package_app)"
+    echo '<?xml version="1.0" encoding="UTF-8"?>' > "$resourceskel"
```

Comment:
If this is code we ever have to look at again, a heredoc might be easier to read, but if this is code we don't touch for years, there's no value in changing it. Editing $LANGUAGE in might be easier. Testing this stuff is such a drag and we've had depressingly little feedback about translations.

Reply:
The skeleton had never changed since the first commit in 2010.